packages/react-topology/src/types.ts

```diff
@@ -171,6 +173,7 @@ export interface Graph<E extends GraphModel = GraphModel, D = any> extends Graph
   getLayout(): string | undefined;
   setLayout(type: string | undefined): void;
   layout(): void;
+  stopLayout(): void;
```

Comment:
I'd prefer to not add this function to the graph api.
They can always do `graph.getLayout()?.stopLayout()`